infra/main.bicep

```diff
@@ -30,31 +30,36 @@ param dockerImage string = 'sswwebsite'
 
 param now string = utcNow('yyyy-MM-ddTHH-mm')
 
-var tags = {
-  // change this to 'core' when you are ready to deploy to production
-  'cost-category': 'dev/test'
+var core = {
+  'cost-category': 'core'
+}
+
+var value = {
+  'cost-category': 'value'
 }
 
 module acr 'acr.bicep' = {
   name: 'acr-${now}'
   params: {
     projectName: projectName
     location: location
-    tags: tags
+    tags: core
   }
 }
 module keyVault 'keyVault.bicep' = {
   name:'keyVault-${now}'
   params: {
     projectName: projectName
     location: location
+    tags:core
```

Comment:
if this is always `core` - then it should be set in the module and not determined outside in the `main` file

Reply:
Done - yes it makes sense